pyscriptjs/src/components/pyrepl.ts

```diff
@@ -216,10 +222,15 @@ export class PyRepl extends BaseEvalElement {
     }
 
     getSourceFromElement(): string {
-        const sourceStrings = [
-            `output_manager.change("` + this.outputElement.id + `")`,
+        let sourceStrings = [
```

Comment:
Hi, maybe this line gets the ESLint error.

```suggestion
        const sourceStrings = [
```

Reply:
But if we change that to const, line 231 won't work I guess.
Thank you for digging into it!

Reply:
You can try it :)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice

Reply:
Thanks! 
Added the commit. Now the build fails, but that is some error in the build process I think.